posthog/session_recordings/queries/session_recording_list_from_filters.py

```diff
@@ -464,7 +465,7 @@ def _where_predicates(self) -> ast.Expr:
 
         (event_where_exprs, _) = self._event_predicates
         if event_where_exprs:
-            exprs.append(ast.Or(exprs=event_where_exprs))
```

Comment:
This is the OR referenced in https://github.com/PostHog/posthog/pull/22955/files#diff-1f84bb4b29839289e9db353424c0a0819c373f08640fd30bbe53561152fe55d8R22